python/deepclient/deep_client.py

```diff
@@ -448,17 +451,222 @@ def default_returning(self, table: str) -> str:
         else:
             return "id"
 
-    async def insert(self):
-        raise NotImplementedError("Method not implemented")
+    async def insert(self, record: Dict, options: Dict = {}) -> Dict:
```

Comment:
https://github.com/deep-foundation/deeplinks/blob/26cb67fd59fcc4f727b80356fa417fcbe53455ea/imports/client.tsx#L682-L700

What is record? As I see in the original source code, we have only options.